### PR TITLE
use StoppableWorkers in testutils/vcamera

### PR DESCRIPTION
**Pull request by penguinland**
This is in test code, so the proof that this works will be that the tests pass, not something I need to try out on actual hardware.

No changes to behavior are intended. This is just simplifying the codebase a little more.

Dan, are you the right person to review this? You and Clyde are the only ones who have touched this directory, and Clyde left the company. but I don't think of you as doing much camera stuff. Feel free to remove yourself and suggest someone else to review instead.

### CODE REVIEW

File: testutils/vcamera/vcamera.go [hunk 1]
```diff
@@ -154,25 +149,21 @@ func startStream(config *Config, dev device) (<-chan struct{}, error) {
 		return nil, err
 	}
 
-	// wait for cmd to finish to avoid leak
-	config.activeBackgroundWorkers.Add(1)
-	go func() {
-		defer config.activeBackgroundWorkers.Done()
+	// Wait for cmd to finish to avoid leak.
+	config.workers.AddWorkers(func(ctx context.Context) {
```
Comment (penguinland) [inline testutils/vcamera/vcamera.go:153]: A note to reviewers: neither the old version nor the new version directly watches the context. However, line 134 starts the command with the context from the StoppableWorkers, so _that's_ where things should cancel early if we call `config.workers.Stop()`.

**Review by randhid — COMMENTED** [review]
Ah I wish you'd talked to the av team prior to starting to work on this - this is one of the files I'm looking at removing wholesale.

**Comment by randhid** [comment]
Not blocking, but one thing - can you follow https://viam.atlassian.net/wiki/spaces/ENG/pages/644481030/Flaky+Test+Reproduction to verify the tests don't know flake? I don't want tech debt added to av's work, they already have a lot.

**Comment by penguinland** [comment]
Rand and I talked: the plan is for this file to be deleted altogether, and she's nervous about adding extra tech debt to the team trying to delete it. I'm going to close without merging, to keep their tasks simpler.

**Pull request closed without merging**